### PR TITLE
Modify "say" and "whisper" so that it sends a whole sentence

### PR DESCRIPTION
**Pull request by GabrielXia**
### Contains

I found today that using command "say" and "whisper" only sends the first word of one sentence. 
A tiny fix to make it send a whole sentence.

- `say()` and `whisper()` receive `String[]` as message
- A small method `join()` transfers `String[]` to `String` with a space between every word
### How to test

- Please press in console `say Hello Terasology!` or `whisper playName Heya good morning :-)`

### CODE REVIEW

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by Cervator** [comment]
Tests out fine - nice and easy! Thanks :-)

Added you to the credits as well

**Comment by GabrielXia** [comment]
Thanks :-)

File: engine/src/main/java/org/terasology/logic/chat/ChatSystem.java [hunk 1]
```diff
@@ -94,25 +94,40 @@ public void onMessage(MessageEvent event, EntityRef entity) {
...
 
         for (EntityRef client : entityManager.getEntitiesWith(ClientComponent.class)) {
-            client.send(new ChatMessageEvent(message, sender.getComponent(ClientComponent.class).clientInfo));
+            client.send(new ChatMessageEvent(messageToString, sender.getComponent(ClientComponent.class).clientInfo));
         }
 
         return "Message sent.";
     }
 
+    private String join(String[] strings, String cement) {
```
Comment (skaldarnar) [inline engine/src/main/java/org/terasology/logic/chat/ChatSystem.java]: @GabrielXia If you are interested in a nice one-line solution for this, using Java 8, have a look at this [Tweet](https://twitter.com/danieldietrich/status/429723665962655744). The code for `join` would look like this (I hope without errors):

```java
private String join(String[] words, String sep) {
  Arrays.stream(words).collect(joining(sep));
}
```

Feel free to open another PR if you want to change this ;-)

**Pull request merged**